document/README.md

```diff
@@ -21,3 +21,65 @@ To build everything and update [webassembly.github.io/spec](https://webassembly.
 make publish
 ```
 Please make sure to only use that once a change has approval.
+
+## Step by step guide to building the spec
+
+Make sure this repository was cloned with `--recursive`:
+
+```
+git clone --recursive https://github.com/WebAssembly/spec
+```
+
+If you have already cloned but without `--recursive`, you can delete and re-clone, or `cd` into `spec` and run:
+
+```
+git submodule update --init --recursive
+```
+
+The rest of these instructions assume you are in the directory where is README is:
+
+```
+cd spec/document
+```
+
+You will need `python3.7`, and `pip`. `pip` should come with Python, if not follow [these installation instructions for `pip`](https://pip.pypa.io/en/stable/installing/), or check your system package manager for `pip3`.
```

Comment:
Maybe split this into subsections like "Perequisites", "Checking Out the Repository", "Building"?